### PR TITLE
Motoman GP70L Support

### PR DESCRIPTION
**Pull request by EricMarcil**
Add support package for Yaskawa/Motoman GP70L robot

### CODE REVIEW

File: motoman_gp70l_support/config/opw_parameters_gp70l.yaml
Comment (gavanderhoorn) [inline motoman_gp70l_support/config/opw_parameters_gp70l.yaml:1]: Could you clarify whether this OPW config was created by you @EricMarcil?

Did you also verify it?

**Review by gavanderhoorn — COMMENTED** [review]
One in-line question.

Something I noticed: the meshes in this PR have the same 'issue' with black parts lacking in contrast as in #535:

image

the cable is much nicer here though.

File: motoman_gp70l_support/config/opw_parameters_gp70l.yaml
Comment (EricMarcil) [reply motoman_gp70l_support/config/opw_parameters_gp70l.yaml:1]: @gavanderhoorn OPW were added by @cjue on PR #493 and #500.  I simply used those as a base and updated the values.  They have not been verified.
I'm not sure if that it something we want to standardize or not.  We can remove the file, if you think it is better not to have them.

**Comment by EricMarcil** [comment]
@gavanderhoorn I started importing model data from our MotoSim software.  That already have color, smaller models and model for cables.
I'll try to adjust the color of the motors...

File: motoman_gp70l_support/config/opw_parameters_gp70l.yaml
Comment (gavanderhoorn) [reply motoman_gp70l_support/config/opw_parameters_gp70l.yaml:1]: I welcome the addition of OPW configuration files. I just want to make sure we have validated the configuration and are sure the values are correct.

**Comment by EricMarcil** [comment]
@gavanderhoorn I fixed the motor color by making it dark gray.  It took a while because every time I tried making changes with blender I would get strange results.  I finally realized that the .dae was actually a text file and edited the color directly in it.
image

**Review by ted-miller — APPROVED** [review]
Approving based on Eric's testing

**Pull request merged**